amis/build_ami.sh

```diff
@@ -117,8 +117,9 @@ check_options() {
     available_arm_os="ubuntu1804 alinux2"  # subset of supported OSes for which ARM AMIs are available
     available_os="centos6 centos7 alinux ubuntu1604 ${available_arm_os}"
     cwd="$(dirname $0)"
-    tmp_dir=$(mktemp -d)
+    tmp_dir=${cwd}/../..
     export COOKBOOK_PATH="${tmp_dir}/cookbook"
```

Comment:
if you set `COOKBOOK_PATH` to be `${cwd}/..`
there is no need to create another cookbook dir and copy the content over again, e.g. you can remove the following code in the do_command function
```
mkdir -p "${COOKBOOK_PATH}"
cp -R ${cwd}/../* "${COOKBOOK_PATH}"
```

Reply:
COOKBOOK_PATH to be `${cwd}/..` met an error, maybe because we can't upload path /.. to tmp/cookbook, so I use `export COOKBOOK_PATH="$(cd ${cwd}/..; pwd)"`